BoxContentSDK/BoxContentSDK/BOXURLSessionManager.m

```diff
@@ -165,7 +177,9 @@ - (NSURLSession *)createProgressSession
 - (NSURLSession *)createBackgroundSessionWithId:(NSString *)backgroundSessionIdentifier maxConcurrentOperationCount:(NSInteger)maxConcurrentOperationCount
 {
     NSURLSessionConfiguration *sessionConfig = [NSURLSessionConfiguration backgroundSessionConfigurationWithIdentifier:backgroundSessionIdentifier];
-
+    if (self.protocolClasses != nil) {
+        sessionConfig.protocolClasses = [self.protocolClasses arrayByAddingObjectsFromArray:sessionConfig.protocolClasses];
```

Comment:
We want to allow setting protocolClasses for both foreground and background. They can both benefit from that in their unit tests.